conf/modules.d/arc.conf

```diff
@@ -26,25 +26,27 @@ arc {
   # If false, messages with empty envelope from are not signed
   allow_envfrom_empty = true;
   # If true, envelope/header domain mismatch is ignored
-  allow_hdrfrom_mismatch = false;
+  allow_hdrfrom_mismatch = true;
   # If true, multiple from headers are allowed (but only first is used)
   allow_hdrfrom_multiple = false;
   # If true, username does not need to contain matching domain
   allow_username_mismatch = false;
   # If false, messages from authenticated users are not selected for signing
-  auth_only = true;
+  auth_only = false;
```

Comment:
I don't think this make any difference as we don't know if the mail is from a authenticated user (ex: `user1@example.com` -> `user2@example.com`),at least that what my testing shows.

Reply:
Never mind, it was a configuration error.